src/opensearch.py

```diff
@@ -61,8 +62,10 @@ def install(self):
             return
 
         try:
+            self._snapd.ensure(snap.SnapState.Latest, channel="latest/stable")
             self._opensearch.ensure(snap.SnapState.Latest, channel="edge")
             self._opensearch.connect("process-control")
+            self._opensearch.connect("shmem-perf-analyzer")
```

Comment:
this can be removed, this interface is `auto-connect`able by default

Reply:
Removed in a new commit.